### ISSUE TITLE
CPack cannot find arduino (win32)

### ISSUE THREAD

**Issue by Ghostkeeper** [Ultimaker/cura-build#25]
Target platform: Windows 32
Host platform: Windows 8 x64

When packaging Cura (after building), I am getting the following error message:

```
[100%] Built target build_exe
Run CPack packaging tool...
CPack: Create package using NSIS
CPack: Install projects
CPack: - Run preinstall target for: cura-build
CPack: - Install project: cura-build
CPack: -   Install component: arduino
CMake Error at C:/Persoonlijk/Projecten/cura-build/build/cmake_install.cmake:39
(file):
  file INSTALL cannot find "C:/Persoonlijk/Projecten/cura-build/arduino".


CPack Error: Error when generating package: Cura
makefile:106: recipe for target 'package' failed
mingw32-make: *** [package] Error 1
```

It seems that CPack, CMake or NSIS can't find arduino. Indeed, the folder is absent. The latest Arduino package is installed. Should something be copied to that folder? Should the CMake target be adapted to include something?

**Comment by diracsbracket** [comment]
Hi,
any update on this? What needs to be done?

**Comment by diracsbracket** [comment]
I just copied the master folder from the arduino zip file on git to the cura-build\arduino folder, and it seems to work...

**Comment by Ghostkeeper** [comment]
Update on this is that the original issue was fixed for a while (but we neglected to close it here on Github). However the Arduino driver installation broke again in the installer a week or two ago. We haven't found the issue yet but we're on it.

Also, the driver installation doesn't work at all on Windows 10 if the user plugged in his printer before installing the driver (or Cura), because Windows 10 installs its own drivers then which are extremely hard to remove. This has been an issue for a long while and there is no easy fix.
